pkg/calculator/adder_test.go

```diff
@@ -12,7 +12,7 @@ import (
 
 func TestAdd(t *testing.T) {
 	t.Parallel()
-	args := []struct {
+	tests := []struct {
 		i, j, expected int
 	}{{
```

Comment:
```suggestion
	type test struct {
		i, j, expected int
	}
	tests := []test{{
```

Reply:
It's pretty idiomatic in go to use anonymous structs for table tests like this, see for example from the (one of the) master(s) himself Dave Cheney (https://dave.cheney.net/2013/06/09/writing-table-driven-tests-in-go).

Reply:
Who actually proposes both :joy:

Reply:
Another example, https://github.com/golang/go/wiki/TableDrivenTests

Reply:
I only introduced it here to enable the different way of passing `tt` below.